include/manifold/manifold.h

```diff
@@ -78,13 +78,10 @@ struct MeshGLP {
   /// This matrix is stored in column-major order and the length of the overall
   /// vector is 12 * runOriginalID.size().
   std::vector<Precision> runTransform;
-  /// Optional: Length NumTri, contains an ID of the source face this triangle
-  /// comes from. When auto-generated, this ID will be a triangle index into the
-  /// original mesh. All neighboring coplanar triangles from that input mesh
-  /// will refer to a single triangle of that group as the faceID. When
-  /// supplying faceIDs, ensure that triangles with the same ID are in fact
-  /// coplanar and have consistent properties (within some tolerance) or the
-  /// output will be surprising.
+  /// Optional: Length NumTri, contains the source face ID this
+  /// triangle comes from. When auto-generated, this ID will be a triangle index
+  /// into the original mesh. This index/ID is purely for external use (e.g.
+  /// recreating polygonal faces) and will not affect Manifold's algorithms.
```

Comment:
One question: should we rename our public `faceID`, since it really has nothing to do with faces now? Maybe `triID` or `triIndex`? 

The worst part is now in our internal `TriRef` struct we have `faceID` which really is a face ID and `tri` which is the internal analogue to this public `faceID`. That's pretty much bound to confuse someone eventually, probably me.

Reply:
Yeah, `triID` sounds better fit here. We should add some documentation for the internal `faceID` to clarify things a bit as well.

Reply:
Since that will be a big change across all of our binding surfaces and such, I'll leave that to its own PR.